Straight_skeleton_2/include/CGAL/create_offset_polygons_from_polygon_with_holes_2.h

```diff
@@ -65,18 +65,19 @@ create_exterior_skeleton_and_offset_polygons_with_holes_2 ( FT const&
                                   );
 }
 
-template<class FT, class OfK, class SsK>
-std::vector< boost::shared_ptr< Polygon_with_holes_2<OfK> > >
+template<class OfK, class C, class SsK, class FT>
+std::vector< boost::shared_ptr< Polygon_with_holes_2<OfK,C> > >
 inline
-create_interior_skeleton_and_offset_polygons_with_holes_2 ( FT const& aOffset, Polygon_with_holes_2<OfK> const& aPoly, SsK const& ssk )
+create_interior_skeleton_and_offset_polygons_with_holes_2 ( FT const& aOffset, Polygon_with_holes_2<OfK,C> const& aPoly, SsK const& ssk )
 {
   return arrange_offset_polygons_2(create_interior_skeleton_and_offset_polygons_2(aOffset,aPoly,ssk));
 }
 
```

Comment:
The patch modifies the order of the template parameters. That is not backward compatible. I do not really understand why the order should be changed. That is probably an error.